WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationFormViewModel.swift

```diff
@@ -92,11 +99,47 @@ final class BlazeCampaignCreationFormViewModel: ObservableObject {
     @Published private(set) var targetLanguageText: String = ""
     @Published private(set) var targetDeviceText: String = ""
 
+    // AI Suggestions
+    @Published private(set) var isLoadingAISuggestions: Bool = true
+    private let storage: StorageManagerType
+    private var product: Product? {
+        guard let product = productsResultsController.fetchedObjects.first else {
+            assertionFailure("Unable to fetch product with ID: \(productID)")
+            return nil
+        }
+        return product
+    }
+
+    @Published private(set) var errorState: ErrorState = .none
```

Comment:
Why don't we use an optional error for simplicity here?

Reply:
We will have to deal with the optional values in the view when we create an Alert [here](https://github.com/woocommerce/woocommerce-ios/blob/feat/11652-integrate-ai-suggestion/WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift#L147). I preferred using this `.none` to avoid that. What do you think?

Reply:
I changed the error handling and made the error optional in 766618731cd2cb55f9ccaa046ce0ac63c63b1603